pkg/oci/client/oci.go

```diff
@@ -195,12 +195,29 @@ func (c *client) FindVolumeAttachment(volumeId string) (core.VolumeAttachment, e
 	return nil, fmt.Errorf("failed to find volume attachment for %q", volumeId)
 }
 
-func (c *client) getAllSubnetsForVNC() (*[]core.Subnet, error) {
+func (c *client) getVCNCompartment() (*string, error) {
+	vcn, err := func() (core.GetVcnResponse, error) {
```

Comment:
Unless I'm missing something the anonymous function is redundant here isn't it?

Reply:
Yes entirely. I forgot to remove when factoring out this stuff into a separate method.